src/content/en/2021/page-weight.md

```diff
@@ -1,21 +1,176 @@
+
+## Analysis
+
+As we post and parse statistical results, the data is often based on transfer sizes. However, we are employing decompressed sizes in this analysis whenever possible. Please note that many of the results listed below are similar or the same as those queries used in 2020.
+
+### Page weight by the numbers
+
+Looking at the page weight on both desktop and mobile devices, the deltas are due to the difference in the amount of transferred resources on the mobile device, the difference is generally small between them.
+
+{{ figure_markup(
+  image="bytes-distribution.png",
+  caption="Distribution of total bytes per page.",
+  description="Bar chart showing the distribution of the total bytes per page. Desktop pages tend to have more bytes throughout the distribution. The 10, 25, 50, 75, and 90th percentiles for mobile pages are: 409, 928, 1,923, 3,749, and 6,890 KB per page.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vThwYNAo2K8eVJVNH_4TuA7IZpG534jXn5vFWphLLuHUo_VNGehDMKoIFWf0GquINRKWNHX66MXUmn9/pubchart?oid=8302830&format=interactive",
+  sheets_gid="895329225",
+  sql_file="bytes_per_type_2021.sql"
+) }}
+
+The key data here shows we are closing in on 6.9 MB of page weight on mobile and 8.1 MB on desktop at the 90th percentile. The data is following an old and consistent trend: growth in page weight is on the upward trajectory that continues from the previous year.
+
+{{ figure_markup(
+  image="bytes-distribution-content-type.png",
+  caption="Median bytes per page by content type.",
+  description="Bar chart showing the median number of bytes per page for images, JavaScript, CSS, and HTML. The median desktop page tends to have more bytes. The median mobile page has 877 KB of images, 470 KB of JS, 66 KB of CSS, and 27 KB of HTML.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vThwYNAo2K8eVJVNH_4TuA7IZpG534jXn5vFWphLLuHUo_VNGehDMKoIFWf0GquINRKWNHX66MXUmn9/pubchart?oid=263743460&format=interactive",
+  sheets_gid="895329225",
+  sql_file="bytes_per_type_2021.sql"
+) }}
+
+A closer inspection shows that the media and average for each resource, and following the trend from years previous to this one, images remain the largest resource followed by JavaScript.
+
+### Requests
+
+As previously explained in this chapter, the number of requests and the type of resource requested has a direct influence on total page weight and page loading performance. The difference between current results for this year and last are notable because it actually shows a tiny decrease in the average number of GET requests. Let's hope that trend continues downward.
```

Comment:
Is the following chart only for GET requests?

When you mention a trend or trajectory, consider showing a chart with the data.